src/index.js

```diff
@@ -96,14 +96,34 @@ const getCheckedRadio = function (nodes, form) {
   }
 };
 
+const escapeCSS =
+  typeof window === 'undefined' ||
+  typeof window.CSS === 'undefined' ||
+  typeof window.CSS.escape !== 'function'
+    ? function (string) {
+        return string;
+      }
+    : window.CSS.escape;
+
 const isTabbableRadio = function (node) {
   if (!node.name) {
     return true;
   }
   const radioScope = node.form || node.ownerDocument;
-  const radioSet = radioScope.querySelectorAll(
-    'input[type="radio"][name="' + node.name + '"]'
-  );
+  let radioSet;
+  try {
```

Comment:
I don't understand what you mean by "safeQuerySelectorAll", and don't follow on the detriment of try..catch.

A modern browser will have CSS.escape and things should just work in that case. Otherwise, they fail, we catch, show a helpful message instead of crashing, and consider the radio not tabbable.

Are you concerned about some performance overhead?

Reply:
In modern browsers escape exist and there is no reason to wrap with try/catch. My concern is for performance, although tbh I'm not sure if try/catch still has any penalty like it use to have.

Reply:
I see. Maybe it did in older browsers, but I'm not concerned for modern ones, and also, if there were any overhead, it would only be on radio buttons. Otherwise, I don't know what else we could do except give up on being nice with the friendly error and just let it crash and trust people on older browsers will read the README note we've added. That behavior would be no different to how it's been for a very long time. But I like the helpful error, personally.

Reply:
OK, I reversed the logic kind of. I think this is what you were getting at. We only `try..catch` if `CSS.escape` is not available. Otherwise, we assume there won't be an exception.

Reply:
I'm moving forward with this.